stablehlo/dialect/VhloOps.td

```diff
@@ -324,6 +324,10 @@ def VHLO_ReplicaIdOpV1 : VHLO_Op<"replica_id"> {
   let results = (outs TensorOf<[UI32]>);
 }
 
+def VHLO_PartitionIdOpV1 : VHLO_Op<"partition_id"> {
```

Comment:
Let's also update [the version log](https://github.com/openxla/stablehlo/blob/29b16547e34ffb119582269a7423be4d44c8e904/stablehlo/dialect/VhloOps.td#L36-L38)?